jekyll/_cci2/webhooks.md

```diff
@@ -54,14 +56,19 @@ Circleci-Signature | When present, this signature can be used to verify that the
 ## Setting up a webhook
 {: #setting-up-a-hook}
 
-Webhooks are set up on a per-project basis. To get started:
+Webhooks are set up on a per-project basis, either within the CircleCI app or via API.
+
+To configure webhooks via API see our documentation for [Webhooks Public API](https://circleci.com/docs/api/v2/#operation/rerunWorkflow).
```

Comment:
Should this link go here: https://circleci.com/docs/api/v2/#tag/Webhook ?

Reply:
I can double check, the engineers put in that link specifically.